libs/features/personalization/personalization.js

```diff
@@ -291,13 +296,44 @@ function getSection(rootEl, idx) {
     : rootEl.querySelector(`:scope > div:nth-child(${idx})`);
 }
 
-function registerCustomAction(cmd, manifestId) {
-  const { action, selector, target } = cmd;
+function registerInBlockActions(cmd, manifestId) {
```

Comment:
What does "InBlock" represent?  Could it just be `registerBlockActions`?

Reply:
`inblock` is going to be a notion adopted by GWP, if you want to rebrand it, I guess it is still not too late.

Reply:
@chrischrischris , @yesil guessed I think.  When authors want to use this feature, they prefix the selector with "in-block".  So I was trying to keep the naming consistent, so it would be more obvious.